uwsgi.ini

```diff
@@ -5,9 +5,11 @@ chmod-socket = 664
 # Graceful shutdown on SIGTERM, see https://github.com/unbit/uwsgi/issues/849#issuecomment-118869386
 hook-master-start = unix_signal:15 gracefully_kill_them_all
 
-module = run:app
-callable = application
+module = run
+callable = app
 need-app = true
+http = 0.0.0.0:8080
+
```

Comment:
Commenting here as I am only able to comment on current changes:

- Is the very large buffer size `buffer-size = 32768g` intended, or would 32768 (32KB) be sufficient?
- You defined a Unix socket (socket = /tmp/uwsgi.sock) and an HTTP socket (http = 0.0.0.0:8080). Are we actually making use of both?
-

Reply:
good point! i reduced the buffer size.